.circleci/config.yml

```diff
@@ -146,14 +170,14 @@ jobs:
       - run:
           name: Build XCFramework
           command: |
-            if [ "${CIRCLE_BRANCH}" == "main" ]; then
+            if [ "${CIRCLE_BRANCH}" == "staging" ]; then
               make archive
             fi
       # verify test app builds
       - run:
           name: Build Test App
           command: |
-            if [ "${CIRCLE_BRANCH}" == "main" ]; then
+            if [ "${CIRCLE_BRANCH}" == "staging" ]; then
```

Comment:
Can you include both "main" and "staging" here?  There are cases where we may merge directly into main so the code may be different from staging, so it is good to rerun this check.